internal/backend/gormkitteh/config.go

```diff
@@ -25,6 +27,18 @@ type Config struct {
 	// If false, the server will fail to start if a migration is required,
 	// and the user has to run 'ak server migrate' explicitly.
 	AutoMigrate bool `koanf:"auto_migrate"`
+
+	Options string `koanf:"options"`
+}
+
+func (c Config) ParseOptions() map[string]string {
```

Comment:
What is the reason options are in this format (`port=8080, level=INFO`) and not an established format line JSON?